### PR TITLE
feat: add sops support

### PR DESCRIPTION
**Pull request by frezbo**
Add sops support.

### CODE REVIEW

File: internal/output/ghworkflow/gh_workflow.go [hunk 1]
```diff
@@ -178,6 +178,32 @@ func (o *Output) AddStep(jobName string, steps ...*Step) {
...
+func (o *Output) AddStepAfter(jobName, afterStepID string, steps ...*Step) {
+	job := o.workflows[ciWorkflow].Jobs[jobName]
+
+	for i, step := range job.Steps {
+		if step.ID == afterStepID {
+			job.Steps = append(job.Steps[:i+1], append(steps, job.Steps[i+1:]...)...)
+
+			break
+		}
+	}
```
Comment (DmitriyMV) [inline internal/output/ghworkflow/gh_workflow.go:204]:
```suggestion
	idx := slices.IndexFunc(job.Steps, func(s *Step) bool { return s.ID == afterStepID })
	if idx != -1 {
		job.Steps = slices.Insert(job.Steps, idx+1, steps...)
	}
```

File: internal/output/ghworkflow/gh_workflow.go [hunk 2]
```diff
@@ -178,6 +178,32 @@ func (o *Output) AddStep(jobName string, steps ...*Step) {
...
+func (o *Output) AddStepBefore(jobName, beforeStepID string, steps ...*Step) {
+	job := o.workflows[ciWorkflow].Jobs[jobName]
+
+	for i, step := range job.Steps {
+		if step.ID == beforeStepID {
+			job.Steps = append(job.Steps[:i], append(steps, job.Steps[i:]...)...)
+
+			break
+		}
+	}
```
Comment (DmitriyMV) [inline internal/output/ghworkflow/gh_workflow.go:191]:
```suggestion
	idx := slices.IndexFunc(job.Steps, func(s *Step) bool { return s.ID == beforeStepID })
	if idx != -1 {
		job.Steps = slices.Insert(job.Steps, idx, steps...)
	}
```

File: internal/project/custom/custom.go [hunk 3]
```diff
@@ -290,7 +291,7 @@ func (step *Step) DroneEnabled() bool {
 // CompileGitHubWorkflow implements ghworkflow.Compiler.
 //
 //nolint:gocognit,cyclop
-func (step *Step) CompileGitHubWorkflow(output *ghworkflow.Output) error {
+func (step *Step) CompileGitHubWorkflow(output *ghworkflow.Output) error { //nolint:gocyclo,maintidx
```
Comment (DmitriyMV) [inline internal/project/custom/custom.go:294]: Why not add to the nolint above?

**Review by DmitriyMV — APPROVED** [review]

File: internal/project/custom/custom.go (hunk 3 above)
Comment (frezbo) [reply internal/project/custom/custom.go:294]: missed that

**Pull request merged**